tests/ut_reporters/ut_teamcity_reporter.ProducesExpectedOutputs.sql

```diff
@@ -3,7 +3,11 @@ declare
   l_output            clob;
   l_expected          varchar2(32767);
 begin
-  l_expected := q'{##teamcity[testSuiteStarted timestamp='%' name='A suite for testing different outcomes from reporters']
+  l_expected := q'{%##teamcity[testSuiteStarted timestamp='%' name='org']
+%##teamcity[testSuiteStarted timestamp='%' name='utplsql']
```

Comment:
@Pazus I'm not sure If this would actually be a valid sequence of messages on TeamCity.
Mainly due to the fact that we have suitepath=`org.utplsql.utplsql.test`.
Maybe teamcity suite name should actually show path? Like it is done in XUnit reporter (see the XUnit test below).